common/reflectivity/reflectivity.cpp

```diff
@@ -185,23 +185,26 @@ void reflectivity::add_depth_sample( float depth_val, int x_in_image, int y_in_i
             _dist_queue.pop_front();
 
         _dist_queue.push_back( dist_r );
-        if (_samples_count < N_STD_FRAMES) _samples_count++;
+        if( _history_size < N_STD_FRAMES )
+            _history_size++;
     }
 }
 
 void rs2::reflectivity::reset_history()
 {
-    if( _samples_count > 0 )
+    if( _history_size > 0 )
     {
         _dist_queue.assign( N_STD_FRAMES, 0 );
-        _samples_count = 0;
+        _history_size = 0;
     }
 }
 
 float rs2::reflectivity::get_samples_ratio() const
 {
-    if (0 != N_STD_FRAMES)
-        return static_cast<float>(_samples_count) / N_STD_FRAMES;
-    else
-        return 0.0f;
+    return static_cast< float >( _history_size ) / N_STD_FRAMES;
+}
+
+bool rs2::reflectivity::is_history_full() const
+{
+    return _history_size / N_STD_FRAMES;
```

Comment:
This looks like a warning...
Why not just simply `return( size() == capacity() );`? It's very easy to understand...

Reply:
Added history_size() and history_capacity() functions